### PR TITLE
Add pages for benchmarking methodology and e2e tests

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
I'm having a bit of an existential crisis about what should be in the cookbook and what should live in supporting repos such as tile-benchmarking. Right now, I'm going by the standard of copy everything required to reproduce the results so that viewers could re-use the framework for their use cases without having to navigate to another repository. This approach also serves to help consolidate and cleanup what is in the tile-benchmarking repo. But I'm open to other approaches + suggestions.

### CODE REVIEW

**Comment by maxrjones** [comment]
this is a good point to bring up! We have been taking a different approach and have been aiming to minimize the amount of duplicated code, such that most of the functionality for benchmarking the direct client approach lives in https://github.com/carbonplan/benchmark-maps and the content here would mostly serve for documentation and visualization of the output (which would be generated by code in the other repo and stored in a public s3 bucket). To make this simpler, we may consider packaging up some of the post-processing code (e.g., functions from https://github.com/carbonplan/benchmark-maps/blob/main/scripts/analysis.py). I think we could strike a balance of explaining everything in enough detail such that only those who really want to dive into to the nitty-gritty parts need to navigate to the other repos, but most of the benchmarking code would still live elsewhere.

**Comment by abarciauskas-bgse** [comment]
That makes sense to me, thanks @maxrjones I will modify this PR accordingly and modify the code in tile-benchmarking to write results to S3.

File: approaches/tiling/e2e-results.ipynb
Comment (abarciauskas-bgse) [inline approaches/tiling/e2e-results.ipynb:1]: This should read results from an S3 storage location

File: approaches/tiling/e2e.ipynb [hunk 1]
```diff
@@ -0,0 +1,233 @@
+{
+ "cells": [
```
Comment (abarciauskas-bgse) [inline approaches/tiling/e2e.ipynb]: This should go back into tile-benchmarking

File: approaches/tiling/gen_test_urls.py [hunk 2]
```diff
@@ -0,0 +1,205 @@
+import fsspec
+import math
```
Comment (abarciauskas-bgse) [inline approaches/tiling/gen_test_urls.py]: This should and other files below should be removed

**Pull request merged**